### ISSUE TITLE
"Forget Wifi" doesn't seem to work

### ISSUE THREAD

**Issue by muesli** [Qrome/marquee-scroller#142]
Clicking "Forget WiFi" the Wemos reboots, but directly connects to the old WiFi again.

**Comment by Qrome** [comment]
What version of the firmware are you running?

**Comment by muesli** [comment]
Didn't work with 2.15, updated to 2.16 just this morning, but no dice.

**Comment by Qrome** [comment]
It is working perfectly for me.   Tell me, have you disabled the administration password for the device?   After clicking on the menu item, you should get a prompt confirming the action, then a basic authentication window where you have to provide the admin / password -- it then reboots and goes into AP mode.

**Comment by Qrome** [comment]
Yeah, it looks like if you turn off the admin password it does not reset.  I will look into it.

**Comment by muesli** [comment]
Thanks! I've tried enabling security credentials for the device now, but it doesn't accept the username/password combination I entered. Guess I have to reset it altogether.

**Comment by scottypres** [comment]
Also having this issue, but using a NodeMCU CP2102 ESP-12E. What was the solution? Even if I reflash the board, it retains memory of the SSID and password.

**Comment by muesli** [comment]
> Also having this issue, but using a NodeMCU CP2102 ESP-12E. What was the solution? Even if I reflash the board, it retains memory of the SSID and password.

Never found a solution for this one, sorry.
